subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -137,36 +217,46 @@ uint8_t ll_cig_parameters_commit(uint8_t cig_id)
 	uint32_t c_max_latency;
 	uint32_t p_max_latency;
 	uint16_t handle_iter;
-	uint8_t  cis_count;
+	uint8_t  num_cis;
+	bool force_framed;
 
 	/* Intermediate subevent data */
 	struct {
 		uint32_t length;
 		uint8_t  total_count;
-	} se[CONFIG_BT_CTLR_CONN_ISO_STREAMS_PER_GROUP];
+	} se[STREAMS_PER_GROUP];
 
-	/* If CIG already exists, controller and host are not in sync */
+	/* If CIG already exists, this is a reconfigure */
 	cig = ll_conn_iso_group_get_by_id(cig_id);
-	LL_ASSERT(!cig);
-
-	/* CIG does not exist - create it */
-	cig = ll_conn_iso_group_acquire();
 	if (!cig) {
-		ll_iso_setup.cis_idx = 0U;
+		/* CIG does not exist - create it */
+		cig = ll_conn_iso_group_acquire();
+		if (!cig) {
+			ll_iso_setup.cis_idx = 0U;
 
-		/* No space for new CIG */
-		return BT_HCI_ERR_INSUFFICIENT_RESOURCES;
+			/* No space for new CIG */
+			return BT_HCI_ERR_INSUFFICIENT_RESOURCES;
+		}
+		cig->state = CIG_STATE_CONFIGURABLE;
+		cig->lll.num_cis = 0;
```

Comment:
There are several instances in this PR where `0` is assigned to a unsigned value instead of `0U`. It unlikely to matter, but just in case that you want to conform to most other zero-assignments

Reply:
Agree